app/jobs/audit/save_to_persistent_storage_job.rb

```diff
@@ -0,0 +1,11 @@
+module Audit
+  class SaveToPersistentStorageJob < ApplicationJob
+    queue_as Audit::Config::JOB_QUEUE_NAME
```

Comment:
I noticed you put the name of the queue in a "namespace module" but I don't think there's a real need for that, in fact it's only used in the tests. If you don't want to repeat the name of the queue, active job classes have a `queue_name` property that returns the name of the queue.

For example:

```ruby
> Users::TouchJob.queue_name
=> "touch_user"
```

In your case you use use `Audit::SaveToPersistentStorageJob.queue_name` in the tests